download.tt

```diff
@@ -0,0 +1,339 @@
+
+    <div class="row">
+      <table class="table">
+        <thead>
+          <tr>
+            <th>NixOS version</th>
+            <th>BLOB URL</th>
+          </tr>
+        </thead>
+        <tbody>
+          [% blobs = JSON.Escape.json_decode(IO.All.new(nixosAzureBlobs).all) %]
+          [% FOREACH version IN blobs.keys.sort; blob = blobs.$version %]
+          <tr>
+            <td><code>[% version %]</code></td>
+            <td><a href="[% blob %]"><code>[% blob %]</code></a></td>
+          </tr>
+          [% END %]
+        </tbody>
+      </table>
+    </div>
+
+    <p>You can create an instance from the command line.
+      For example, to create an instance of version
+      <code>[% blob = blobs.$latestNixOSSeries; latestNixOSSeries %]</code>
+      of NixOS using azure-cli, just run:</p>
+
+    <pre class="well">
+# settings
+# vm name must be lowercase and letters/numbers only because it's also used as the domain name
+export VM_NAME="nixosvm"
+export RESOURCE_GROUP="rg-nixosvm"
+export LOCATION="eastus"
+export STORAGE_ACCOUNT="nixosvm"
+export SOURCE_URI="<code>[% blob %]</code>"
+
+# create group
+az group create \
+    --name "$RESOURCE_GROUP" \
+    --location "$LOCATION
+
+# create storage
+az storage account create \
+    --name "$STORAGE_ACCOUNT" \
+    --group "$RESOURCE_GROUP" \
+    --location "$LOCATION" \
+    --sku Standard_LRS
+
+az storage account keys list \
+    --name "$STORAGE_ACCOUNT" \
+    --group $RESOURCE_GROUP
+
+# export one of the storage access keys
+export STORAGE_ACCOUNT_KEY="..."
+echo $STORAGE_ACCOUNT_KEY
+
+az storage container create \
+    --account-name "$STORAGE_ACCOUNT"
+    --account-key "$STORAGE_ACCOUNT_KEY"
+    --name vm-images
+
+# copy the image blob
+az storage blob copy start \
+    --source-uri "$SOURCE_URI" \
+    --destination-blob "nixos-unstable-nixops.vhd" \
+    --destination-container "vm-images" \
+    --account-name "$STORAGE_ACCOUNT" \
+    --account-key "$STORAGE_ACCOUNT_KEY"
+
+# wait for blob copy status to change pending->success
+az storage blob show \
+    --name "nixos-unstable-nixops.vhd" \
+    --container-name "vm-images" \
+    --account-name "$STORAGE_ACCOUNT" \
+    --account-key "$STORAGE_ACCOUNT_KEY"
+
+
+# create vm
+az vm create \
+    --name "$VM_NAME" \
+    --resource-group "$RESOURCE_GROUP" \
+    --location "$LOCATION" \
+    --public-ip-address-dns-name "$VM_NAME" \
+    --image "https://$STORAGE_ACCOUNT.blob.core.windows.net/vm-images/nixos-unstable-nixops.vhd" \
+    --storage-account "$STORAGE_ACCOUNT" \
+    --storage-container-name "$VM_NAME" \
+    --use-unmanaged-disk \
+    --os-disk-size-gb "32" \
+    --os-type linux \
+    --admin-username "azureuser" \
+    --generate-ssh-keys
+
+# show machine properties and log in
+az vm show \
+    --name "$VM" \
+    --resource-group "$GROUP"
+
+ssh azureuser@"$VM_NAME"."$LOCATION".cloudapp.azure.com</pre>
+  </section>
+  <section class="tab-pane" id="nixos-google">
```

Comment:
this is completely outdated, so it don't make sense to review.

don't call 18.09 latest!!!1

add a warning for now WHY this is the case and best link to an issue with status

Reply:
For now I'm leaving it as it is. I have this on my todo list to fix it for next release.

Reply:
ah i ment to say i'm going to leave the warning but i have no ticket to link to, yet